app/components/case-studies-components/CaseStudyBody.tsx

```diff
@@ -0,0 +1,38 @@
+import { WorkProcessSymbolType } from "@/types";
+import {
+  CaseStudyHeader,
+  CaseStudyTechStack,
+  CaseStudyProblemStatement,
+  CaseStudyChallengesAndLearnings,
+  SimilarCaseStudies,
+} from ".";
+import { GetInTouchCard } from "../homepage-components";
+import { CaseStudyType } from "@/utils/interfaces";
```

Comment:
If CaseStdyType is a "type," it should go into your types folder, not the interfaces. 
`import { WorkProcessSymbolType, CaseStudyType } from "@/types";`

Reply:
@cdsalty Good catch. I've changed this here and changed all of the necessary imports